commands.adoc

```diff
@@ -223,10 +223,18 @@ Similarly, you can use `$ vagrant service-manager status openshift`,
 
 If you need to install the client binary for a specified service, use: +
 
-`vagrant service-manager install-cli [service]` +
+`vagrant service-manager install-cli [service] [options] [arg]` +
 
 The possible options for service are `docker` and `openshift`. +
 
+The following options can be used: +
+
+`--cli-version`::  Version of binary to install +
+`--path`::   Absolute or relative path where to install the binary +
+`-h, --help`::   Prints help for the specific command being run.
+
+_arg_ represents the valid value for the respective `--cli-version` and/or `--path` options.
+
```

Comment:
This is the only place we are using Italics for '_arg_'. I would prefer we use `[arg]` to keep it consistent with the style and also to prevent confusion for users. However, whichever style we decide to use, it would have to be applied to the entire document for words like, commands, sub-commands, options etc. Or we can simply change _arg_ to arg.